types/axiosTokenInterceptor.d.ts

```diff
@@ -0,0 +1,18 @@
+declare module 'axios-token-interceptor' {
+  import { AxiosRequestConfig } from 'axios';
+
+  type InterceptorOptions = {
+    header?: string,
```

Comment:
also, please check eslint for .d.ts files
after string must be `;` not `,`

Reply:
The `;` symbol actual for classes or interface. In the types should be `,` symbol.